x-pack/plugins/cross_cluster_replication/public/app/services/auto_follow_pattern_validators.test.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import { validateAutoFollowPattern } from './auto_follow_pattern_validators';
+
+jest.mock('ui/index_patterns/index_patterns.js', () => ({
+  IndexPatternsProvider: jest.fn(),
+}));
+
+jest.mock('ui/index_patterns/index_patterns_api_client_provider.js', () => ({
+  IndexPatternsApiClientProvider: jest.fn(),
+}));
+
+describe('Auto-follow pattern validators', () => {
+  describe('validateAutoFollowPattern()', () => {
+    it('returns empty object when autoFollowPattern is undefined', () => {
+      const errors = validateAutoFollowPattern();
+      expect(errors).toMatchSnapshot();
```

Comment:
I removed the other assertions from the tests because I believe the snapshot covers them already.